src/Fixer/FunctionNotation/NativeFunctionInvocationFixer.php

```diff
@@ -0,0 +1,285 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\FunctionNotation;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException;
+use PhpCsFixer\Fixer\ConfigurableFixerInterface;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ */
+final class NativeFunctionInvocationFixer extends AbstractFixer implements ConfigurableFixerInterface
+{
+    /**
+     * @var string[]
+     */
+    private $configuration = array();
+
+    /**
+     * @var array
+     */
+    private static $defaultConfiguration = array(
+        'exclude' => array(),
+        'include' => array(),
+    );
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null)
+    {
+        if (null === $configuration) {
+            $this->configuration = self::$defaultConfiguration;
+
+            return;
+        }
+
+        foreach ($configuration as $key => $value) {
+            if (!\array_key_exists($key, self::$defaultConfiguration)) {
+                throw new InvalidFixerConfigurationException(
+                    $this->getName(),
+                    \sprintf(
+                        '"%s" is not handled by the fixer.',
+                        $key
+                    )
+                );
+            }
+
+            foreach ($configuration[$key] as $functionName) {
+                if (!\is_string($functionName) || \trim($functionName) === '' || \trim($functionName) !== $functionName) {
+                    throw new InvalidFixerConfigurationException(
+                        $this->getName(),
+                        \sprintf(
+                            'Each element must be a non-empty, trimmed string, got "%s" instead.',
+                            \is_object($functionName) ? \get_class($functionName) : \gettype($functionName)
+                        )
+                    );
+                }
+            }
+        }
+
+        if (empty($configuration)) {
+            throw new InvalidFixerConfigurationException(
+                $this->getName(),
+                'Configuration must define "exclude" and/or "include" as an array.'
+            );
+        }
+
+        if (\array_key_exists('include', $configuration) && \array_key_exists('exclude', $configuration)) {
+            $contradictingFunctions = \array_intersect(
```

Comment:
Not sure about this one, because actually, to determine contradictions, the function names should be normalized. Should we even care about contradicting configurations?